handbook/engineering/distribution/recurring_processes.md

```diff
@@ -135,7 +152,7 @@ As with everyone at Sourcegraph, we join the [weekly company meeting](https://ab
 
 ### Weekly Distribution team sync
 
-Mon @ 2pm PST we hold an [internal
+Mon @ 11am PST we hold an [internal
 team sync](https://docs.google.com/document/d/1otP6F8qfm2yNOW1hjTszkkuiYF1MGp31s5ATeA76ij4/edit) via Zoom. It usually does not involve product managers or higher ups. The goal is to:
```

Comment:
```suggestion
team sync](https://docs.google.com/document/d/1otP6F8qfm2yNOW1hjTszkkuiYF1MGp31s5ATeA76ij4/edit) via Zoom. The goal is to:
```

Please also delete the "### Distribution management sync" section since that meeting doesn't exist anymore.